src/plugins/intel_npu/src/plugin/npuw/compiled_model.cpp

```diff
@@ -341,7 +343,14 @@ ov::npuw::CompiledModel::CompiledModel(const std::shared_ptr<ov::Model>& model,
     }      // for(orderedSubgraphs)
 
     std::map<std::size_t, std::string> forced_sub_devices{};
-    const std::string fsd_opt = m_cfg.get<::intel_npu::NPUW_SUBMODEL_DEVICE>();
+    std::string fsd_opt = m_cfg.get<::intel_npu::NPUW_SUBMODEL_DEVICE>();
+    // Change "last" keyword to tail subgraph number
+    std::size_t last_pos = fsd_opt.find("last");
+    if (last_pos != std::string::npos) {
+        fsd_opt.erase(last_pos, 4);
+        fsd_opt.insert(last_pos, std::to_string(last_sub_idx));
```

Comment:
Will this cause issue with index > 4 symbols? However, this is not the case for us. But might be add a comment here?

Reply:
Why should it be a problem? 
We're deleting here substring "last" (so it has length 4). Maybe better to use separate variable for "last" substring to avoid hardcode here.